libkbfs/folder_branch_ops.go

```diff
@@ -1947,8 +1947,8 @@ func (fbo *folderBranchOps) unembedBlockChanges(
 		}
 		fblock, ok := block.(*FileBlock)
 		if !ok {
-			return nil, false,
-				errors.Errorf("Block for %s is not a file block", ptr)
+			return nil, false, errors.Errorf(
+				"Block for %s is not a file block. Block type: %T", ptr, block)
```

Comment:
I'd prefer avoiding sentences in these, maybe: `Block for %s is not a file block, block type: %T` instead?